subgraph/src/utils/transaction.ts

```diff
@@ -0,0 +1,10 @@
+import { Transaction } from "@amxx/graphprotocol-utils/generated/schema";
+import { ethereum } from "@graphprotocol/graph-ts";
+
+export default function log(event: ethereum.Event): Transaction {
```

Comment:
same comment as for /subgraph/src/utils/event.ts...

This is already available in `@amxx/graphprotocol-utils/src/transactions.ts`